app/livechat/client/views/app/tabbar/chatRoomHistoryItem.html

```diff
@@ -0,0 +1,26 @@
+<template name='chatRoomHistoryItem'>
+	<li class='list-chat' id='{{_id}}' aria='{{v.token}}' >
+		<div class="history-div">
+			<div class="visitor-img">
+				<img src="/avatar/{{responseBy.username}}" alt="">
+			</div>
+			<div class="visitor-history-details">
+				<div class="visitor-details">
+					<h1>{{responseBy.username}}</h1>
+						{{#if open}}
+							<p class='open'>Open</p>
+						{{else}}
+							<p>Close at time {{time}}</p>
+						{{/if}}
+				</div>
+				<div class="total-messages">
+					<p>{{room.msgs}} messages</p>
+				</div>
+				<div class="agent-comment">
+					<p class="comment-heading">Agent comment:</p>
+					<p class="comment-text">"{{lastMessage.msg}}"</p>
```

Comment:
The `lastMessage.msg` is not the message that will be displayed here. The closing room message needs to be fetched from the DB, so you will need a helper method to return the message to the template.
```suggestion
					<p class="comment-text">"{{closingRoomMessage}}"</p>
```

Reply:
performed a check if the lastMesaage is livechat-close message only then it will be displayed